x-pack/plugins/security_solution/public/common/components/exceptions/exceptionable_endpoint_fields.json

```diff
@@ -1,19 +1,11 @@
 [
   "Endpoint.policy.applied.id",
-  "Target.process.Ext.code_signature.status",
```

Comment:
Question: will the removal of these cause a problem for any existing environment where these might have already been used in entries that are stored? do we need migration?

Reply:
no, because the lists are still equivalent.  This file now represents the common default fields for Exceptions that appear in the modal.  These are now moved to the Mac/Windows only Exception field defaults.

Here's how they're combined: https://github.com/elastic/kibana/blob/master/x-pack/plugins/security_solution/public/common/components/exceptions/helpers.tsx#L59

Also, the Linux exceptions are new for `7.14`

Reply:
crap - I forgot Linux is new in this release 🤦‍♂️ - never mind.